tests/CMakeLists.txt

```diff
@@ -1035,6 +1053,10 @@ add_cmdline_test(stlexport             EXPERIMENTAL OPENSCAD SUFFIX stl FILES ${
 add_cmdline_test(manifold-stlexport    EXPERIMENTAL OPENSCAD SUFFIX stl FILES ${EXPORT_STL_TEST_FILES} EXPECTEDDIR stlexport ARGS --enable=predictible-output --enable=manifold --render)
 add_cmdline_test(objexport             EXPERIMENTAL OPENSCAD SUFFIX obj FILES ${EXPORT_OBJ_TEST_FILES} ARGS --render --enable=predictible-output)
 add_cmdline_test(3mfexport             EXPERIMENTAL OPENSCAD SUFFIX 3mf FILES ${EXPORT_3MF_TEST_FILES} ARGS --enable=predictible-output)
+add_cmdline_test(offexport             EXPERIMENTAL OPENSCAD SUFFIX off FILES ${EXPORT_OFFEXPORT_TEST_FILES} ARGS --enable=predictible-output --render)
+add_cmdline_test(color-offexport       EXPERIMENTAL OPENSCAD SUFFIX off FILES ${EXPORT_OFFEXPORT_TEST_FILES} ARGS --enable=manifold --enable=render-colors --enable=predictible-output --render)
+add_cmdline_test(color-offimport       EXPERIMENTAL OPENSCAD SUFFIX png FILES ${EXPORT_OFFIMPORT_TEST_FILES} ARGS --enable=render-colors)
```

Comment:
This doesn't looks right: The test names indicate OFF export/import, but you're just doing a straight-up OpenSCAD render to PNG. To test an export/import round-trip, use `EX_IM_PNGTEST_PY`

Reply:
shuffled the tests a bit, hope it makes more sense now